Database installation on Windows (MySQL / MySQLi) and clearing cache

**Describe the bug**
When you want to install Kimai2 on a WAMP machine, then you run into this error:
`[ERROR] Failed to create database: An exception occurred in driver: could not find driver`

That is because MySQL**i** is the default driver on Windows and the database connection is
_DATABASE_URL=mysql://db_user:db_password@127.0.0.1:3306/db_name_

When you change this to _DATABASE_URL=**mysqli**://db_user:db_password@127.0.0.1:3306/db_name_ then the installation works fine until you come to "Clearing the cache for the prod environment with debug false":

`[ERROR] Failed to clear cache: Unsupported database engine: mysqli. Kimai only supports one of: mysql, sqlite`

Afterwards you get `[OK] Congratulations! Successfully installed Kimai 2 version 1.12 (stable)`

**To Reproduce**
1. Enter `php bin\console kimai:install -n` to the cmd prompt on a fresh Windows installation
2. Change the _DATABASE_URL to mysqli in .env
3. Enter `php bin\console kimai:install -n` again

**Logfile**
No logfile has been created on installation.

**Additional context**
 - Kimai version: 1.12 (Git-HEAD from 2020-12-11)
 - PHP version: 7.4.11
 - Device: Windows 10
 - Database: 10.5.4-MariaDB

There is a different error message now:
` [ERROR] Failed to clear cache: Could not find config file for database engine. Looked at
         D:\Daten.Web\kimai2-test\config\packages\doctrine/mysqli.yaml`

I don't know whether it would work when I copy mysql.yaml to mysqli.yaml and replace everything in there with **mysqli**.

How can I get to the state before executing "bin\console kimai:install -n"?

--- a/src/DependencyInjection/Compiler/DoctrineCompilerPass.php
+++ b/src/DependencyInjection/Compiler/DoctrineCompilerPass.php
@@ -22,8 +22,9 @@ class DoctrineCompilerPass implements CompilerPassInterface
      * @var string[]
      */
     private $allowedEngines = [
-        'mysql',
-        'sqlite'
+        'mysql' => 'mysql',
+        'mysqli' => 'mysql',
+        'sqlite' => 'sqlite',
     ];
 
     private function getEnvVar(string $name): ?string
@@ -72,14 +73,14 @@ class DoctrineCompilerPass implements CompilerPassInterface
             );
         }
 
-        if (!\in_array($engine, $this->allowedEngines)) {
+        if (!\array_key_exists($engine, $this->allowedEngines)) {
             throw new \Exception(
                 'Unsupported database engine: ' . $engine . '. Kimai only supports one of: ' .
-                implode(', ', $this->allowedEngines)
+                implode(', ', array_keys($this->allowedEngines))
             );
         }
 
-        return $engine;
+        return $this->allowedEngines[$engine];
     }
 
     /**

